### PR TITLE
Resolve issue 25896: Cannot create folder using only letters

### PR DESCRIPTION
**Pull request by edenduong**
### Description (*)

1. Consistency between message and behavior when create Non-latin letter folder => Change message to:
`Please rename the folder using only Latin letters, numbers, underscores and dashes.`

This PR was covered by Unit Test.

### Fixed Issues (if relevant)

1. magento/magento2#25896: Cannot create folder using only letters

### Manual testing scenarios (*)

1. On CMS edit page open media gallery
2. Click "Create Folder"
3. Type in "папка"
4. Click "OK"

#### Expected result

1. Consistency between message and behavior. => Change message to:
`Please rename the folder using only Latin letters, numbers, underscores and dashes.`

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by dmytro-ch** [comment]
@magento give me test instance

**Comment by magento-engcom-team** [comment]
Hi @dmytro-ch. Thank you for your request. I'm working on Magento instance for you

**Comment by magento-engcom-team** [comment]
Hi @dmytro-ch, here is your [new Magento instance](https://pr-25904.instances.magento-community.engineering).
Admin access: [https://pr-25904.instances.magento-community.engineering/admin](https://pr-25904.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`

File: app/code/Magento/Cms/Model/Wysiwyg/Images/Storage.php [hunk 1]
```diff
@@ -416,7 +416,7 @@ public function createDirectory($name, $path)
     {
         if (!preg_match(self::DIRECTORY_NAME_REGEXP, $name)) {
             throw new \Magento\Framework\Exception\LocalizedException(
-                __('Please rename the folder using only letters, numbers, underscores and dashes.')
+                __('Please rename the folder using only Latin letters, numbers, underscores and dashes.')
```
Comment (dmytro-ch) [inline app/code/Magento/Cms/Model/Wysiwyg/Images/Storage.php:419]: Could you please also add the new translation string to `Magento_Cms/i18n/en_US.csv`?

**Review by dmytro-ch — CHANGES_REQUESTED** [review]
It looks like extending the media browser component `lib/web/mage/adminhtml/browser.js` (`newFolder` function) with frontend validation is complicated. So let's stick to at least proper backend validation messages for now.

**Comment by edenduong** [comment]
@dmytro-ch: I have updated the csv file. You can check it again. Thank you !

**Review by dmytro-ch — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @dmytro-ch, thank you for the review.
[ENGCOM-6443](https://jira.corp.magento.com/browse/ENGCOM-6443) has been created to process this Pull Request

**Comment by engcom-Alfa** [comment]
:heavy_check_mark: QA Passed

**Pull request merged**